GitCommands/DiffMergeTools/Kdiff3.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+
+namespace GitCommands.DiffMergeTools
+{
+    internal class Kdiff3 : DiffMergeTool
+    {
+        /// <inheritdoc />
+        public override string ExeFileName => "kdiff3.exe";
+
+        /// <inheritdoc />
+        public override string Name => "kdiff3";
+
+        /// <inheritdoc />
+        public override IEnumerable<string> SearchPaths => new[]
+        {
+            // regkdiff3path
```

Comment:
Registry does not seem to be queried any more - maybe OK

Reply:
Ah yes, I missed this...
Do we really want t bring it back?

Reply:
It is a functionality change, can probably be documented. KDiff3 is probably not the tool most install nowadays (it was a good addition when it was added for sure).

Reply:
Noted the removal in the commit message